events/2023-10-in-person/index.qmd

```diff
@@ -80,4 +80,126 @@ some topics and less on others.
+    - It was really useful to see how they work as well, though maybe not loving the purely scrum style they use
+- Hiring them and what/how they can work with us
+    - Could either hire them after we have a core functional box or hire them to help use develop our process and workflow (getting started, project management, teamwork practices, "how to work" rather than "what to work on")
+    - If hired for process, we'll need to give them clear requirements for what we want from them
+        - We might need some time to sit down and define the requirements for designing the how to work together
+    - [X] Action: To contact them about hiring them, what are the next steps? How them charge
+    - [ ] Action: Need to spend some time writing doc on how to work together, the process. This could be the next “focus stage”, on starting to write out a design document
+    - [ ] Action: Make a detailed list of what outputs we would like from them
+        - E.g. training material or documentation on working together and their process
+        - E.g. finalize the design doc (maybe another review of it)?
+- Things we could take from them
+    - Their approach to visualizing things (e.g. for instance with the vertical lines)
+        - For virtual settings: Look into online tools that help with doing the visualize
+    - Making sure everything is visible (can’t talk about things that are invisible)
+    - Clearer and more descriptive "events"
+    - Having time during meetings to dig into "user stories"/tasks
+        - Including adding some time estimates
+        - Every certain frequency, like every month
+        - Use them as the basis for a "focus" period
+    - Flow
+        - Keep in mind the overall roadmap
+        - Have a week of planning and deciding tasks
+        - Then have a period of working on those tasks over the next month
+    - [ ] Action: Add these potential items / workflows to our team docs
+- Misc
+    - We could include the timeline as part of our documentation
+
+### Update on progress, barriers and challenges
+
+- Getting lost down learning rabbit hole
+- Not having a clear path forward on what to work on
+- Feeling a bit managed vs led
+- Trust needs building up
+
+### Next steps to push progress forward
+
+- Moving forward, tracking progress, and building trust
+    - Building monthly sprints, with a period of roadmapping in those days
+    - Use Kanban board with constrained time periods to use for the focus phases, with dedicated time pre-phase to describe and plan together what needs to be done and how long each task takes
+    - Focus on smaller focused periods of working, with iterative development/feedback and smaller units of roadmapping.
+    - [ ] Action: Schedule time shortly 
+    - Have twice-weekly update meetings
+    - Limit external meeting
+    - Make use of Issue’s
+        - Issue templates for how to structure or fill in Issue/task
+        - [ ] Action: Richard make the first draft of template
```

Comment:
Oh! I have created a first draft in the .github repo for a user story issue template